src/prpy/base/robot.py

```diff
@@ -317,14 +320,26 @@ def postprocess_trajectory(traj, kw_args):
             else:
                 return traj
 
-        # Perform postprocessing on a future trajectory.
-        def defer_trajectory(traj_future, kw_args):
-            return postprocess_trajectory(traj_future.result(), kw_args)
-
         # Return either the trajectory result or a future to the result.
         if 'defer' in kw_args and kw_args['defer'] is True:
-            from trollius.executor import get_default_executor
-            executor = kw_args.get('executor') or get_default_executor()
-            return executor.submit(defer_trajectory, result, kw_args)
+            import trollius
+
+            # Perform postprocessing on a future trajectory.
+            @trollius.coroutine
+            def defer_trajectory(traj_future, kw_args):
```

Comment:
I thought we mostly moved away from Trollius coroutines to Trollius futures. Is this an artifact of this branch being out of date? Or are we still using coroutines in some places?